src/harshadBJun20/CountOfCharsUsingWrapper.java

```diff
@@ -0,0 +1,45 @@
+package harshadBJun20;
+
+/*Assignment 17 : 25th Jun 2020
+
+1) Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2*/
+
+public class CountOfCharsUsingWrapper{
+	int digitCount,uppercaseCount,lowercaseCount,specialCharCount;
+	void printCountOfChars(String str1) {
+		
+		for(int index=0;index<str1.length();index++){
+			
+			if(Character.isDigit(str1.charAt(index))==true)
+				digitCount++;  			
+			if(Character.isUpperCase(str1.charAt(index))==true)
+				uppercaseCount++;
+			if(Character.isLowerCase(str1.charAt(index))==true)
+				lowercaseCount++;	
```

Comment:
Here so many comparisons happening, which is not required. 
Like if char is digit so it will go in your all if block. which is wrong right !!
So here you need to write if, else if....else block.